CONTRIBUTING.md

```diff
@@ -191,6 +191,13 @@ Improvements must have the title "[Area or module] : [Added/Refactored/Improved/
 
 New collectors must have the title "[New collector] : [Name of collector]".
 
+#### Commit messages when PRs are merged
+
+When a PR gets squashed and merged into master, the title of the commit message (first line) must be the PR title
+followed by the PR number.
```

Comment:
> followed by the PR number

Why the PR number is needed?

Reply:
> this is the default behaviour of "Squash and Merge" 

oh, right

Reply:
So it is easy to find the discussion about the work.

Reply:
i think it is impossible to exclude PR number if you squash the PR 🤔

Reply:
🙈 possible ofc, PR num appended by defult when we squash, just checked 🙈

Reply:
😀

Reply:
I _literally_ think we're just documenting what is default behaviour here and expectations :)